model/user_test.go

```diff
@@ -143,7 +143,7 @@ func TestUserIsValid(t *testing.T) {
 
 	user.LastName = strings.Repeat("a", 64)
 	user.Position = strings.Repeat("a", 128)
-	require.Error(t, user.IsValid())
+	require.Nil(t, user.IsValid())
```

Comment:
This was false-passing before, as `Error`, but now that is checking for `Nil` is having the expected behavior.